Adyen.IntegrationTest/BaseTest.cs

```diff
@@ -151,10 +156,16 @@ protected AdjustAuthorisationRequest CreateAdjustAuthorisationtestRequest(string
         
         protected Client CreateApiKeyTestClient()
         {
-            var apikey = ClientConstants.Xapikey;
-            return new Client(apikey, Model.Enum.Environment.Test);
-        }
-        
+            var config = new Config()
+            {
+                XApiKey = ClientConstants.Xapikey,
+                Environment =  Environment.Test
+            };
+            // create basic factory
+            var factory = new ServiceCollection().AddHttpClient().BuildServiceProvider().GetRequiredService<IHttpClientFactory>();
```

Comment:
I noticed that you're instantiating a new service collection and getting a real httpclientfactory here, I recommend just passing a new HttpClient() here to get a similar effect so you do not need to include `using Microsoft.Extensions.DependencyInjection` the DI dependencyInjection extensions packages.

What do you think?

Reply:
Fair point, it was just to test the overloaded constructor with the factory really. Since we know it works we might as well remove the DI extension though.